.github/workflows/build-go.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         run: go run schemas/main.go --schema-next=false
 
       - name: Run generation
-        run: kiota generate -l go --ll trace -o $(pwd)/../go-sdk/github/octokit -n "github.com/octokit/go-sdk/github/octokit/" -d schemas/downloaded.json
+        run: kiota generate -l go --ll trace -o $(pwd)/../go-sdk/octokit/github -n "github.com/octokit/go-sdk/octokit/github/" -d schemas/downloaded.json
```

Comment:
I liker this line of thinking for sure - I amgrappling with  `go-sdk/octokit` feeling a bit redundant - it's almost like saying SDK SDK (kit).  But I like this approach and I could just be over thinking it.  

Perhaps this `go-sdk/octokit` + the `github` sub folder that represents models and APIs is more like `bucket/parts`.

Approved the PR, just some ramblings that we're in my head. Where ever we land I'll make sure the .NET SDK migrates to that patten as well.

Reply:
I hear your thoughts on redundancy; that definitely bothers me as well. One way around this would be to use a `pkg` or `src` directory instead of Octokit. So then we'd have something that looks like: 

```
go.mod
go.sum
LICENSE
pkg/
README.md

./pkg:
authentication/
github/
headers/

./pkg/authentication:
request.go
token_provider.go
token_provider_test.go

./pkg/github:
advisories/
api_client.go
app/
applications/
appmanifests/
...
```

How does that look?

Reply:
(We've discussed over Slack and we're going with `pkg` for Go and `src` for .NET).